packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -41,15 +41,30 @@ type CallbackConfigType = {|
 
 export type CallbackIdType = CallbackConfigType;
 
-import requestAnimationFrameForReact from 'shared/requestAnimationFrameForReact';
 import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 
+import warning from 'fbjs/lib/warning';
+
+if (__DEV__) {
+  if (
+    ExecutionEnvironment.canUseDOM &&
+    typeof requestAnimationFrame !== 'function'
+  ) {
+    warning(
+      false,
+      'schedule depends on requestAnimationFrame. Make sure that you load a ' +
```

Comment:
"schedule depends on requestAnimationFrame" doesn't make it obvious "schedule" is a name so it reads a bit odd. Maybe we can reword the whole message to not mention the library name? e.g.

>This browser doesn't support requestAnimationFrame. Make sure [...]